### ISSUE TITLE
VirusTotal V2 observable query not working

### ISSUE THREAD

**Issue by tsale** [intelowlproject/IntelOwl#299]
Hello,

I believe there is a problem with the VirusTotal V2 observable query. If you try to use the V2 VirusTotal to search for an observable, it will not work. You basically cannot choose to use the V2 version on the IntelOwl web interface and returns no results if you try to scan with all available analyzers.

I am using the latest version of IntelOwl v1.9.1.

**Comment by CITIZENDOT** [comment]
I'll try to resolve this.

**Comment by mlodic** [comment]
Hey @tsale, that was an intended change that we added in the v.[1.8.0](https://github.com/intelowlproject/IntelOwl/blob/master/.github/CHANGELOG.md#v180).

The problem was that all those variants of VirusTotal analysis were available by default so, if a user would have liked to go for the option `run_all_available_analyzers` he would have used many different VT analyzers, would have burned VT quota and and would have found unexpected results.

So, we decided to remove some options that were available by default, VT version 2 analysis too. If you prefer to use the old APIs, we suggest to enable those analyzers again [here](https://github.com/intelowlproject/IntelOwl/blob/master/configuration/analyzer_config.json#L1009) by changing the `disable` flag to `true`. This would work even without restarting IntelOwl server.
